static-webapps/slate-portfolio-manager/src/components/sidebar/NewLevelPanel.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div
+    :class="`level-panel -new mb-2 cbl-level-${Level}`"
+  >
+    <b-container class="bg-cbl-level-25">
+      <b-row class="py-2 align-items-center">
+        <b-col>
+          <h3 class="h6 m-0 text-muted">
+            Year {{ Level }}
+          </h3>
+        </b-col>
+        <b-col cols="4">
+          <button
+            class="btn btn-primary btn-sm float-right"
+            @click="createLevel"
+          >
+            Enroll
+          </button>
+        </b-col>
+      </b-row>
+    </b-container>
+  </div>
+</template>
+
+<script>
+import client from '@/store/client';
+import emitter from '@/store/emitter';
+
+export default {
+  props: {
+    Level: {
+      type: Number,
+      default: () => null,
+    },
+    StudentID: {
+      type: Number,
+      default: () => null,
+    },
+    CompetencyID: {
+      type: Number,
+      default: () => null,
+    },
+  },
+  methods: {
+    createLevel() {
+      const { StudentID, CompetencyID, Level } = this;
+      const data = [
+        {
+          StudentID,
+          CompetencyID,
+          Level,
+          EnteredVia: 'enrollment',
+          ID: -1,
+          Class: 'Slate\\CBL\\StudentCompetency',
+          BaselineRating: null,
+        },
+      ];
+      const url = 'cbl/student-competencies/save';
```

Comment:
@chriscauley this fails under hosted builds, I didn't think to test it out in the PR environment build. I think we need a leading `/` here—I'm gonna try that out and just go ahead and commit and finish the release if it works, just tagging you for future reference

![Screen Shot 2022-11-29 at 6 54 28 PM](https://user-images.githubusercontent.com/458494/204674140-364e6e58-1558-4f61-bac1-84074fdc77fb.png)